File: readability/readability.c
#include <cs50.h>
#include <stdio.h>
#include <string.h>
#include <math.h>
#include <ctype.h>

int main(void)
{
    string text = get_string("Text: ");

    int letters = 0;
    for (int i = 0; i < strlen(text); i++)
    {
        if (islower(text[i]))
        {
        letters++;
        }
        else if (isupper(text[i]))
        letters++;
    }
    printf("%i letters\n", i);
}

File: sentimental-readability/readability.py
from cs50 import get_string
from math import round
from stdio import isalpha

text = get_string("Text: ")

letters = 0
words = 1
sentences = 0

for i in range(len(text)):
    if isalpha(text[i]):
        letters += 1
    elif text[i] == " ":
        words += 1
    elif text[i] == "." or text[i] == "!" or text[i] == "?":
        sentences += 1
